KIPs/kip-xx.md

```diff
@@ -0,0 +1,52 @@
+---
+kip: to be assigned
+title: Transaction data gas cost reduction
+author: Lake (@hyunsooda) and Ollie (@blukat29)
+discussions-to: https://github.com/kaiachain/kips/issues/22
+status: Draft
+type: Standards Track
+category: Core
+created: 2024-10-18
+---
+
+## Simple Summary
+
+Changing to calldata gas cost.
+
+## Abstract
+
+This KIP introduces a new tag for calldata gas payment, defining a calculation method that distinguishes between zero and non-zero bytes, with distinct costs assigned to each byte.
```

Comment:
```suggestion
Reduce the calldata gas costs to the EIP-2028 level. The purpose is to achieve better compatibility with Ethereum-oriented SDKs and tools.
```

Reply:
Do you think this should be moved to the 'Motivation' section, or is a motivation section unnecessary?

Reply:
Motvation added. https://github.com/kaiachain/kips/pull/23/commits/469fe0e96e917924559d5a419175d154326a68f8